### PR TITLE
Replace `upload-dif` to `debug-files upload`

### PR DESCRIPTION
**Pull request by denrase**
## :scroll: Description

Closes https://github.com/getsentry/sentry-dart-plugin/issues/93

## :green_heart: How did you test it?

Updated tests.

## :pencil: Checklist

- [x] I reviewed submitted code
- [x] I added tests to verify changes
- [ ] I updated the docs if needed
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: lib/sentry_dart_plugin.dart [hunk 1]
```diff
@@ -62,7 +62,7 @@ class SentryDartPlugin {
 
     _setUrlAndTokenAndLog(params);
 
-    params.add('upload-dif');
+    params.add('debug-files upload');
```
Comment (marandaneto) [inline lib/sentry_dart_plugin.dart:65]: > ❯ sentry-cli debug-files --help
> Locate, analyze or upload debug information files.
> 
> Usage: sentry-cli debug-files [OPTIONS] <COMMAND>
> 
> Commands:
>   bundle-sources  Create a source bundle for a given debug information file
>   check           Check the debug info file at a given path.
>   find            Locate debug information files for given debug identifiers.
>   print-sources   Print source files linked by the given debug info file.
>   upload          Upload debugging information files.
>   help            Print this message or the help of the given subcommand(s)
> 
> Options:
>       --header <KEY:VALUE>       Custom headers that should be attached to all requests
>                                  in key:value format.
>       --auth-token <AUTH_TOKEN>  Use the given Sentry auth token.
>       --log-level <LOG_LEVEL>    Set the log output verbosity. [possible values: trace, debug, info,
>                                  warn, error]
>       --quiet                    Do not print any output while preserving correct exit code. This
>                                  flag is currently implemented only for selected subcommands.
>                                  [aliases: silent]
>   -h, --help                     Print help

I guess `debug-files` goes before `_setUrlAndTokenAndLog` and then `upload`, you can try locally as well.

**Comment by marandaneto** [comment]
Renamed the docs https://github.com/getsentry/sentry-docs/pull/7091

**Comment by marandaneto** [comment]
@denrase can you just do a quick test that everything works as expected? by enabling every feature of the plugin and be sure that no command is broken with a simple empty app.

**Comment by denrase** [comment]
@marandaneto Just ran `getsentry/sentry-dart/flutter/example` with all options enabled and pointing locally to the current checked out branch. Ran without errors as far as i could tell.

Bildschirmfoto 2023-06-12 um 11 18 52

**Pull request merged**